index.html

```diff
@@ -20,7 +20,8 @@
   <link rel="stylesheet" href="css/normalize.css">
   <link rel="stylesheet" href="css/skeleton.css">
   <link rel="stylesheet" href="css/style.css">
-
+  <noscript>
+    <link rel="stylesheet" href="css/noscript.css"></noscript>
```

Comment:
also is there a way to do this without using a separate css file, can we define onlywithjs in style.css and then add in something like a media query that checks whether js is enabled